internal/cmd/machine_create.go

```diff
@@ -112,25 +112,54 @@ var createCommand = &cobra.Command{
 			if err := viper.UnmarshalKey("sites", &sites); err != nil {
 				return err
 			}
-			for _, site := range sites {
-				// check the site.Path for a tilde
-				if strings.HasPrefix(site.Path, "~/") {
-					home, _ := homedir.Dir()
-					site.Path = strings.Replace(site.Path, "~", home, 1)
+
+			// check for mounts
+			var mounts []config.Mount
+			if viper.IsSet("mounts") {
+				if err := viper.UnmarshalKey("mounts", &mounts); err != nil {
+					return err
 				}
-				mountAction, err := action.Mount(name, site.Path, site.Domain)
-				if err != nil {
-					siteErrs = append(siteErrs, err)
-					continue
+
+				for _, m := range mounts {
+					// check for the tilde
+					sourceDir := m.Source
+					if strings.HasPrefix(m.Source, "~/") {
```

Comment:
The home directory is just one example... this should be a relative (and maybe absolute?) path too, no?

Reply:
for portability, we should use ~ but we can definitely add other examples but I think ~ is the most common.